internal/cmd/service/update/tag/tag.go

```diff
@@ -0,0 +1,125 @@
+package tag
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/zeabur/cli/internal/cmdutil"
+	"github.com/zeabur/cli/internal/util"
+)
+
+type Options struct {
+	id   string
+	name string
+
+	environmentID string
+
+	tag string
+
+	skipConfirm bool
+}
+
+func NewCmdTag(f *cmdutil.Factory) *cobra.Command {
+	opts := &Options{}
+	zctx := f.Config.GetContext()
+
+	cmd := &cobra.Command{
+		Use:   "tag",
+		Short: "Update image tag of a prebuilt service",
+		PreRunE: util.RunEChain(
+			util.NeedProjectContextWhenNonInteractive(f),
+			util.DefaultIDNameByContext(zctx.GetService(), &opts.id, &opts.name),
+			util.DefaultIDByContext(zctx.GetEnvironment(), &opts.environmentID),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runUpdate(f, opts)
+		},
+	}
+
+	util.AddServiceParam(cmd, &opts.id, &opts.name)
+	util.AddEnvOfServiceParam(cmd, &opts.environmentID)
+	cmd.Flags().BoolVarP(&opts.skipConfirm, "yes", "y", false, "Skip confirmation")
+	cmd.Flags().StringVarP(&opts.tag, "tag", "t", "", "The new tag of the image")
+
+	return cmd
+}
+
+func runUpdate(f *cmdutil.Factory, opts *Options) error {
+	if f.Interactive {
+		return runInteractive(f, opts)
+	} else {
+		return runNonInteractive(f, opts)
+	}
+}
+
+func runInteractive(f *cmdutil.Factory, opts *Options) error {
+	zctx := f.Config.GetContext()
+
+	if _, err := f.ParamFiller.ServiceByNameWithEnvironment(zctx, &opts.id, &opts.name, &opts.environmentID); err != nil {
+		return err
+	}
+
+	varInput, err := f.Prompter.Input("Enter a new image tag", "latest")
```

Comment:
Is there any API to list all supported tags? If so, we can use `Prompter.Select`

Reply:
> Is there any API to list all supported tags? If so, we can use `Prompter.Select`

It's hard to say ... Not every image registry has a public API to query tags